src/api/generate-xlsx/index.ts

```diff
@@ -50,6 +51,13 @@ const XLSX = (application: Application): Promise<string> => {
         worksheet.addRow(row);
       });
 
+      /**
```

Comment:
could this not just be the `//` comment to save space?

Reply:
Good shout, i've updated all the comments in this file